pkg/client/conditions.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
This is an unfortunate place, since it won't work with all of our various clients...